get-pants.sh

```diff
@@ -1,221 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
i think we want to keep this - firstly for local usage (im using it regularly) - but also if you check the pants action it looks for this file and downloads it if its not present